framework/wazuh/utils.py

```diff
@@ -136,6 +136,14 @@ def check_sort_fields(allowed_sort_fields, sort_by):
             incorrect_fields = ', '.join(sort_by - allowed_sort_fields)
             raise WazuhException(1403, 'Allowed sort fields: {0}. Fields: {1}'.format(', '.join(allowed_sort_fields), incorrect_fields))
 
+    def get_subkeys_values(input_dict, field):
+        # Return value for nested fields if '_' is find, else return not nested field value
+        aux = field.split('_')
```

Comment:
What happens if two fields are nested using '_' and one of them have a '_' character? For instance, 'parent_pre_field'. That split would return an array with 3 elements 'parent', 'pre', 'field'. I'd rather use split with param maxsplit=1 as explained in https://docs.python.org/3.7/library/stdtypes.html#str.split

Reply:
I add the param `maxsplit` and we fix the structure in 4.0